huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,534 @@
+package lts
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API LTS DELETE /v3/{project_id}/lts/access-config
+// @API LTS POST /v3/{project_id}/lts/access-config
+// @API LTS PUT /v3/{project_id}/lts/access-config
+// @API LTS POST /v3/{project_id}/lts/access-config-list
+func ResourceCceAccessConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCceAccessConfigCreate,
+		UpdateContext: resourceCceAccessConfigUpdate,
+		ReadContext:   resourceCceAccessConfigRead,
+		DeleteContext: resourceHostAccessConfigDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: cceAccessConfigResourceImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				Description: `Specifies the name of the cce access. The name consists of 1 to 64 characters.
```

Comment:
```suggestion
				Description: `The name of the CCE access. The name consists of 1 to 64 characters.
```
No `Specifies`, and not `cce`, should be `CCE`.

Reply:
done